composer.json

```diff
@@ -17,8 +17,8 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "container-interop/container-interop": "^1.1",
         "psr/http-message": "^1.0",
+        "psr/container": "^1.0",
```

Comment:
Hm... we can't change namespaces in expressive 1.X to `Psr\Container ... ` it will be BC break.

I think we can do it with v2 (as it is not yet released), because container-interop 1.2 extends psr-container, so for example ServiceManager with container-interop 1.2 is actually PSR-11 compatible.